documentation-samples/quickstarts/Face/FaceQuickstart/FaceQuickstart.csproj

```diff
@@ -0,0 +1,70 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
if you have to check-in csproj (which I believe you don't). Please consider to use .NET core csproj format.

Reply:
Thanks @yangyuan  I deleted it.